README.md

```diff
@@ -65,4 +65,9 @@ Ideally, the new branch should be based on the latest `dev` branch.
 1. Reviewer and contributor may have discussions back and forth until all comments addressed.
 1. Wait for the pull request to be merged.
 
+### Remove a bundle
+
+If an existing bundle needs to be removed, please follow the steps mentioned above to do the delete manipulations during the created new branch in your fork, and then create a new pull request.
+However, bundles that are already stored in the release will not be changed if the corresponding source code is removed.
+
```

Comment:
Hi @wyli and @Nic-Ma , as for the instructions, I wrote like this in the readme, what do you think?

Reply:
thanks, looks good to me, perhaps add: "Please create an issue or contact the repository maintainers if a released bundle should be removed permanently."